core/state_transition.go

```diff
@@ -591,6 +601,9 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 			ReturnData: ret,
 		}, nil
 	}
+
+	st.refundOperatorCost()
```

Comment:
Would be good to add a comment above explaining why we call this _before_ state gas refunds are applied, and why we give the operator fee reward to the operator _after_. And again wondering, is this intended? I don't see a reply to my other questions regarding this yet.

Reply:
This comment still needs to be added above 
```go
	if rules.IsOptimismIsthmus {
		st.refundOperatorCost()
	}
```
like 
> Calling st.refundOperatorCost() after st.gasRemaining is updated above, so that state refunds are taken into account when calculating operator fees.

Reply:
Done